sagemaker-python-sdk/tensorflow_keras_cifar10/tensorflow_keras_CIFAR10.ipynb

```diff
@@ -0,0 +1,213 @@
+    "utils.cifar10_download()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Upload the data to a S3 bucket"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "inputs = sagemaker_session.upload_data(path='/tmp/cifar10_data', key_prefix='data/DEMO-cifar10')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "**sagemaker_session.upload_data** will upload the CIFAR-10 dataset from your machine to a bucket named **sagemaker-{region}-{*your aws account number*}**, if you don't have this bucket yet, sagemaker_session will create it for you."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Complete source code\n",
+    "- [cifar10_cnn.py](cifar10_cnn.py): Keras model"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "scrolled": false
+   },
+   "outputs": [],
+   "source": [
+    "!cat cifar10_cnn.py"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Create a training job using the sagemaker.TensorFlow estimator"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "scrolled": false
+   },
+   "outputs": [],
+   "source": [
+    "from sagemaker.tensorflow import TensorFlow\n",
+    "\n",
+    "\n",
+    "estimator = TensorFlow(entry_point='cifar10_cnn.py',\n",
+    "                       role=role,\n",
+    "                       hyperparameters={'min_eval_frequency': 10},\n",
+    "                       training_steps=1000, evaluation_steps=100,\n",
+    "                       train_instance_count=1, train_instance_type='ml.c4.xlarge')\n",
+    "\n",
+    "estimator.fit(inputs)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "collapsed": true
+   },
+   "source": [
+    "**Atention**: Keras modules have a known issue and cannot be issue for distributed (multi-instances) training.\n",
+    "Keep **train_instance_count** == 1 until they the TensorFlow/Keras team, support this feature.\n",
+    "More information about the issue: https://github.com/tensorflow/tensorflow/issues/14504\n",
+    "\n",
+    "\n",
+    "# Deploy the trained model to prepare for predictions\n",
+    "\n",
+    "The deploy() method creates an endpoint which serves prediction requests in real-time."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "predictor = estimator.deploy(initial_instance_count=1, instance_type='ml.m4.xlarge')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Making some predictions"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Creating fake prediction data\n",
```

Comment:
Is there a reason you're doing prediction on fake data?  Why not use some records from the dataset itself?  This might be a little bit easier for the user to follow.